### DISCUSSION TITLE
Support for DayZ (SA)?

### DISCUSSION THREAD

**Discussion by Eutyches** [muttleyxd/arma3-unix-launcher#192] [Ideas]
I greatly appreciate the Arma 3 Unix Launcher. Installing the pre-packaged binary from the AUR and using GloriousEggroll's Proton build to run Arma itself makes everything smooth and pleasant. Thank you to everyone who works on this, that is, @muttleyxd and others.

In the past few months, DayZ stand-alone has shown considerable progress in the ProtonDB: https://www.protondb.com/app/221100

Even connecting to modded servers is possible via command line parameters or the impressive (but certainly not 'beginner friendly') DZTUI tool: https://github.com/aclist/dztui

Since DayZ is another Bohemia Interactive title that uses very similar launch parameters and mod configuration to Arma 3, and even a launcher that is nearly identical (presumably, it was an in-house 'copy and paste' job), I wonder if you (muttleyxd) or someone else with the requisite skill might be able to port the Arma 3 Unix Launcher to DayZ SA, whether as a fork or as an extension of this current project in some way.

**Comment by muttleyxd** [comment]
Somewhat resolved by #195. My launcher still does not feature any kind of server browser, but I was able to connect to few servers with it.
